errata: edit to docker warning (#1476)
meilisearch/documentation: learn/getting_started/quick_start.md

Data written to a **Docker container is not persistent** and is wiped every time the container is stopped. We recommend using a shared Docker volume between containers and host machines to provide persistent storage.

> On macOS and Windows, you must be careful not to mount volumes from the host to the container because I/O operations between the filesystems will be very slow. Instead make sure the mounted volumes remain inside the docker vm. If this is not an option, we recommend using the native application or a [cloud-hosted option](#cloud-deploy).

Reviewer: ```suggestion
On macOS and Windows, do not mount volumes from the host to the container—this will make I/O operations between the filesystems very slow. Instead make sure the mounted volumes remain inside the docker vm. If this is not an option, we recommend using the native application or a [cloud-hosted option](#cloud-deploy).
```

Author: is the dash between container and this on purpose?

Reviewer: Yeah, it's an [emdash](https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use#:~:text=The%20em%20dash%20can%20function,upon%20something%20that%20precedes%20it.), one of my favorite forms of punctuation 😄 

Feel free to replace it with a colon or semicolon if you prefer!
